src/topology/algebra/group/basic.lean

```diff
@@ -1071,22 +1075,21 @@ begin
 end
 
 @[to_additive]
-lemma topological_group.t3_space [t1_space G] : t3_space G := ⟨⟩
+lemma topological_group.t3_space [t0_space G] : t3_space G := ⟨⟩
 
 @[to_additive]
-lemma topological_group.t2_space [t1_space G] : t2_space G :=
+lemma topological_group.t2_space [t0_space G] : t2_space G :=
 by { haveI := topological_group.t3_space G, apply_instance }
 
 variables {G} (S : subgroup G) [subgroup.normal S] [is_closed (S : set G)]
 
 @[to_additive]
-instance subgroup.t3_quotient_of_is_closed
-  (S : subgroup G) [subgroup.normal S] [is_closed (S : set G)] : t3_space (G ⧸ S) :=
+instance subgroup.regular_quotient_of_is_closed
```

Comment:
Why was this renamed?

Reply:
This was a leftover change from before the merge.